lib/trino-orc/pom.xml

```diff
@@ -101,6 +101,12 @@
         </dependency>
 
         <!-- for testing -->
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-hadoop-toolkit</artifactId>
```

Comment:
There's no reason to worry about this for tests, since tests don't use class loader isolation. Creating a new library just for this one factory seems overkill. We could simply add a suppression here.

Reply:
i agree this is an overkill for tests only. in trino-orc this is test-only, but in trino-rcfile and raptor it's a compile-time depdendency. i wouldn't want to make copies of the code for rcfile and raptor